lte/gateway/c/session_manager/UpfMsgManageHandler.cpp

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*****************************************************************************
+  Source      	UpfMsgManagerHandler.cpp
+  Version     	1.0
+  Date       	2020/02/17
+  Product     	SessionD
+  Subsystem   	UPF converged Landing object in SessionD
+  Author/Editor Venu Kumar Gurrapu
+  Description 	Acts as 5G Landing object in SessionD & start 5G related flow
+*****************************************************************************/

```

Comment:
remove these lines and stick with the copyright statement.

Reply:
Removed, kept copyright  lines alone.